README.md

```diff
@@ -1 +1,145 @@
-# fe-hometask
\ No newline at end of file
+# React & Rockets
+
+**Description:** Challenge for Adjust Frontend Developer
+
+**Owner:** [adjust](https://github.com/adjust)
+
+**Contacts:** frontend-hiring@adjust.com
+
+---
+
+## Welcome to our little coding exercise! 👋
+
+Here you will be given the opportunity to play with JavaScript and Rockets in the same project. For this, we recommend you to block 2-3 hours of your time to complete it.
+
+**Please make sure you have read this page entirely, before starting the challenge**. If you have any doubts, please reach out to us by [opening an issue](#how-to-request-help) as described at the bottom of this page.
+
+**Importantly**, we would like to give all candidates taking this test the same opportunity to solve the exercise in their own way. Because of this, **please do not fork or share this repo (nor your solution) with anybody 🙏🏻**
+
+<img align="center" src="https://i.imgur.com/ekyJNd9.jpg" width="600">
+
+## Instructions
+
+1. Push your solution to a **private repo** in your **personal Github account**
+2. When you are ready for us to take a look, please add [adjust-frontend-hiring][adjust-frontend-hiring] (GitHub user) as collaborator
+
+## General notes
+
+- The final solution must be a working React application.
+  - It's up to you how to bootstrap a new React application.
+- You can use any third-party library if you see a good fit for it.
+- Bringing in best practices such as tests, type safety, linting is appreciated.
+- It's not necessary to heavily comment the code, please only leave comments where you think they are necessary to understand the related code.
+- Aspects to be assessed:
+  - Code Style – meaningful naming, reasonable conciseness and consistency with applied code style conventions (i.e. sticking to either single or double quotes when declaring strings, but not mixing).
+  - Quality Assurance – the solution you upload should work as expected, conforming to the specifications of the tasks.
```

Comment:
I tried to elaborate on "aspects to be assessed" to make them more transparent (we could probably add a note about ES6 usage?), although I also considered if we could completely remove this section. What do you think?

Reply:
good to have imo: candidates my be able to optimise for different metrics - time to solve, code readability, reusability and so on. Giving them a hint what we would like to check should be helpful for us to understand if they can write code in such manner

Reply:
Updated the wording to be:

> "Code Style – meaningful naming conventions, consistent code formatting, and emphasis on code readability and future extensibility."

Let me know if that works, otherwise I'll resolving for now.